src/components/MDXContainer.js

```diff
@@ -36,6 +36,8 @@ const defaultComponents = {
       <img
         width={props.width ? props.width : 'inherit'}
         src={props.src}
+        // this prevents images from stretching when the width is set to a percent value
+        className={props.width ? 'unbound' : props.className}
```

Comment:
hmm would it be better have `unbound` in addition to any className passed in?

```suggestion
        className={cx(props.className, props.width && 'unbound')}
```